app/Controllers/admin/ServerInfo.php

```diff
@@ -116,4 +117,23 @@ private function buildVersionInfo()
         }
         $this->view->getTplVarManager()->set('versionInfo', $versionInfo);
     }
+    
+    private function buildDatabaseInfoSection ()
+    {
+        $databaseInfo = [];
+        $system = zen_get_system_information();
+        $databaseInfo['heading'] = TITLE_DATABASE_VARIABLES . $system['db_version'] . ($system['mysql_strict_mode'] == true ? '<em> ' . TITLE_MYSQL_STRICT_MODE . '</em>' : '');
+        
+        $databaseInfo['fields'] = [];
+        $show_variables = $this->dbConn->Execute ("SHOW VARIABLES");
+        while (!$show_variables->EOF) {
```

Comment:
Since v1.5.5 we now support, and prefer, a `foreach` syntax when looping thru db queries, instead of the `while( !EOF) ... MoveNext...` syntax.

So this would look more like:

```php
        $results = $this->dbConn->Execute ("SHOW VARIABLES");
        foreach($results as $result) {
            $databaseInfo['fields'][] = array (
                'name' => $result['Variable_name'],
                'value' => zen_not_null ($result['Value']) ? $result['Value'] : '&nbsp;'
            );
        }
```